src/TypeScript/BuildTypeScript.ps1

```diff
@@ -0,0 +1,7 @@
+
+# TODO:  Expand on this.
+cd Sarif.Sdk
+npm install
```

Comment:
obviously, no error checking here. we have lots of examples that provide basic structure for erroring out. and so, i'd go grab that pattern rather than checking in nothing at all (since the incidental costs of stealing existing code is low)